tests/functional/test_application_restricted.py

```diff
@@ -0,0 +1,232 @@
+import datetime
+import jwt
+import uuid
+import time
+import requests
+from .config_files import config
```

Comment:
Ideally we should use a package import instead of a relative import, but it's ok for this to slide for now